glue_solar/core/sunpy_maps/loader.py

```diff
@@ -28,6 +28,8 @@ def __init__(self, directory):
         self.cancel.clicked.connect(self.reject)
         self.ok.clicked.connect(self.finalize)
 
+        print(directory)
```

Comment:
```suggestion
```

Reply:
Hmm... Actually this is a trick I have been using to avoid an unexpected error regarding `directory` path. I will remove this soon.